build.gradle

```diff
@@ -131,8 +134,13 @@ jar {
 task functional (dependsOn: ['testJar']) << {
     exec {
         executable = 'java'
-        args = ['-jar', testJar.archivePath, 'migrateSchema']
+        args = ['-jar', '-Dcassandra.storagedir=/tmp/cassandra', testJar.archivePath, 'migrateSchema']
+
     }
 }
 
 check.dependsOn(functional)
+
+test {
+    jvmArgs '-Dcassandra.storagedir=/tmp/cassandra'
```

Comment:
It's a shame this is needed as you have to add it for every test run in IntelliJ. I think it has been fixed by this issue and PR https://github.com/jsevellec/cassandra-unit/issues/176 but it's not yet been released with this fix.